Tests/UserspaceEmulator/test-run-ls.cpp

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
Typo in commit title, something like ``Tests: Assure that UserspaceEmulator does not regress via `ue ls` ``?

Reply:
I needed to abbreviate UserspaceEmulator, but yes verbs might be a good idea